iset.mm

```diff
@@ -117423,8 +117486,619 @@ Infinite sums (cont.)
+      nnred nnge1d clt wb nngt0d ledivmul syl112anc mpbird cseq cli cdm efcllem
+      caddc syl cvv seqex 1red difrp mpbid rpreccld cabs absidd eqbrtrd breldmg
+      crp geolim mp3an2i isumle efval expcl isumclim eqcomd 3brtr4d ) AHBFUAZIJ
+      ZXAUBKZLJZFUCZHXBFUCZBUDKZMMBUEJZLJZNAXDXBFGHBGUAZIJZXJUBKLJUFZGHXKUFZOHU
+      GAUHZABUIPZXAHPZXAXLKXDQABCUJZBGXLXAXLUQZUKRABSPZXPXDSPCBXAULRAXPUMZXPXBS
+      PZXAXMKXBQAXPUNZXTBXAAXSXPCUOZYBUPGXAXKXBHSXMXJXABIURXMUQUSUTZAXSXPYACBXA
+      VARZXTXDXBNTZXBXCXBVBJNTZXTXBXCYEXTXCXPXCVCPAXAVDVEZVHZXTBXAYCYBAOBNTXPDU
+      OVFXTXCYHVIVGXTYAYAXCSPOXCVJTYFYGVKYEYEYIXTXCYHVLXBXBXCVMVNVOAXOVTXLOVPVQ
+      VRZPXQBGXLXRVSWAVTXMOVPZWBPAXIWLPYKXIVQTYKYJPVTXMOWCAXHABMVJTZXHWLPZEAXSM
+      SPYLYMVKCAWDBMWEUTWFWGABFXMXQABWHKBMVJABCDWIEWJYDWMZYKXIWBWLVQWKWNWOAXOXG
+      XEQXQBFWPWAAXFXIAXBXIFXMOHUGXNYDAXOXPXBUIPXQBXAWQRYNWRWSWT $.
+  $}
+
+  ${
+    $d x A $.
+    $( Value of the sine function.  (Contributed by NM, 14-Mar-2005.)  (Revised
+       by Mario Carneiro, 10-Nov-2013.) $)
+    sinval $p |- ( A e. CC -> ( sin ` A ) =
+    ( ( ( exp ` ( _i x. A ) ) - ( exp ` ( -u _i x. A ) ) ) / ( 2 x. _i ) ) ) $=
+      ( vx cc wcel ci cmul co ce cfv cneg cmin cdiv csin wceq mulcld efcl oveq2
+      a1i syl fveq2d ax-icn negicn subcld 2mulicn cc0 cap wbr 2muliap0 divclapd
+      c2 id cv oveq12d oveq1d df-sin fvmptg mpdan ) ACDZEAFGZHIZEJZAFGZHIZKGZUJ
+      EFGZLGZCDAMIVFNURVDVEURUTVCURUSCDUTCDUREAECDURUARURUKZOUSPSURVBCDVCCDURVA
+      AVACDURUBRVGOVBPSUCVECDURUDRVEUEUFUGURUHRUIBAEBULZFGZHIZVAVHFGZHIZKGZVELG
+      VFCCMVHANZVMVDVELVNVJUTVLVCKVNVIUSHVHAEFQTVNVKVBHVHAVAFQTUMUNBUOUPUQ $.
+
+    $( Value of the cosine function.  (Contributed by NM, 14-Mar-2005.)
+       (Revised by Mario Carneiro, 10-Nov-2013.) $)
+    cosval $p |- ( A e. CC -> ( cos ` A ) =
+       ( ( ( exp ` ( _i x. A ) ) + ( exp ` ( -u _i x. A ) ) ) / 2 ) ) $=
+      ( vx cc wcel ci cmul co ce cfv caddc cdiv ccos wceq a1i mulcld efcl oveq2
+      c2 syl fveq2d cneg ax-icn id negicn addcld halfcld cv oveq1d df-cos mpdan
+      oveq12d fvmptg ) ACDZEAFGZHIZEUAZAFGZHIZJGZRKGZCDALIUTMUMUSUMUOURUMUNCDUO
+      CDUMEAECDUMUBNUMUCZOUNPSUMUQCDURCDUMUPAUPCDUMUDNVAOUQPSUEUFBAEBUGZFGZHIZU
+      PVBFGZHIZJGZRKGUTCCLVBAMZVGUSRKVHVDUOVFURJVHVCUNHVBAEFQTVHVEUQHVBAUPFQTUK
+      UHBUIULUJ $.
+  $}
+
+  $( Domain and codomain of the sine function.  (Contributed by Paul Chapman,
+     22-Oct-2007.)  (Revised by Mario Carneiro, 30-Apr-2014.) $)
+  sinf $p |- sin : CC --> CC $=
+    ( vx cc ci cv cmul co ce cfv cneg cmin c2 cdiv csin df-sin wcel ax-icn mpan
+    mulcl efcl syl negicn subcld cc0 cap 2mulicn 2muliap0 divclap mp3an23 fmpti
+    wbr ) ABBCADZEFZGHZCIZUKEFZGHZJFZKCEFZLFZMANUKBOZUQBOZUSBOZUTUMUPUTULBOZUMB
+    OCBOUTVCPCUKRQULSTUTUOBOZUPBOUNBOUTVDUAUNUKRQUOSTUBVAURBOURUCUDUJVBUEUFUQUR
+    UGUHTUI $.
+
+  $( Domain and codomain of the sine function.  (Contributed by Paul Chapman,
```

Comment:
cosine

Reply:
Fixed in set.mm and iset.mm.  Funny that this one went unfixed for (apparently 15) years.